Tools/TestWebKitAPI/Tests/WebKitCocoa/ProcessSwapOnNavigation.mm

```diff
@@ -6526,6 +6526,103 @@ function saveOpenee()
     done = false;
 }
 
+TEST(ProcessSwap, NavigateCrossOriginWithOpenerWithRestrictedOpenerTypeNoOpener)
+{
+    auto processPoolConfiguration = psonProcessPoolConfiguration();
```

Comment:
Please use smart pointers explicitly instead of hiding them behind `auto` in new code. Same applies below.
e.g. `RetainPtr processPoolConfiguration `